MATURITY.md

```diff
@@ -83,18 +83,18 @@ Mentors and community members are encouraged to contribute to this page and comm
 
 ## Independence
 
-| ID   | Description                                                                              | Status                                                                                                                                                                                                                                                                                                             |
-|------|------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| IN10 | The project is independent from any corporate or organizational influence.               | :white_large_square: The project team gathers people from different companies (AsiaInfo, bilibili, China Mobile, eBay, iQIYI, NetEase, T3, Zhihu ...). No company or organization has significantly more influence than any other. We can note a growth of the contributions coming from different contributors.   |
-| IN20 | Contributors act as themselves, not as representatives of a corporation or organization. | :white_check_mark: The contributors act on their own initiative without representing a corporation or organization.                                                                                                                                                                                                |
+| ID   | Description                                                                              | Status                                                                                                                                                                                                                                                                                                            |
+|------|------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| IN10 | The project is independent from any corporate or organizational influence.               | :white_check_mark: The project team gathers people from different companies (AsiaInfo, bilibili, China Mobile, eBay, iQIYI, NetEase, T3, Zhihu ...). No company or organization has significantly more influence than any other. We can note a growth of the contributions coming from different contributors.    |
```

Comment:
There are 12 PPMC members come from different companies, including AWS, eBay, Huawei, Sensors Data, NetEase, China Mobile, Ctrip, and T3
https://whimsy.apache.org/roster/ppmc/kyuubi

And the contributors is growing.
https://github.com/apache/incubator-kyuubi/graphs/contributors